ecosystem/typescript/sdk_v2/src/utils/api-endpoints.ts

```diff
@@ -0,0 +1,27 @@
+export const NetworkToIndexerAPI: Record<string, string> = {
+  mainnet: "https://indexer.mainnet.aptoslabs.com/v1/graphql",
+  testnet: "https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql",
+  devnet: "https://indexer-devnet.staging.gcp.aptosdev.com/v1/graphql",
+};
+
+export const NetworkToNodeAPI: Record<string, string> = {
+  mainnet: "https://fullnode.mainnet.aptoslabs.com/v1",
+  testnet: "https://fullnode.testnet.aptoslabs.com/v1",
+  devnet: "https://fullnode.devnet.aptoslabs.com/v1",
+  local: "http://localhost:8080/v1",
+};
+
+export const NetworkToFaucetAPI: Record<string, string> = {
+  mainnet: "https://faucet.mainnet.aptoslabs.com",
+  testnet: "https://faucet.testnet.aptoslabs.com",
+  devnet: "https://faucet.devnet.aptoslabs.com",
+  local: "http://localhost:8081",
+};
+
+export enum Network {
+  MAINNET = "mainnet",
+  TESTNET = "testnet",
+  DEVNET = "devnet",
+  LOCAL = "local",
+  CUSTOM = "custom",
+}
```

Comment:
Does this include a `toString()` and a `fromString()` kind of stuff?

I found myself having to do `toString() to compare sometimes to inputs from the wallet adapter wallets.

Reply:
not completely sure what you mean.. you can do `Network.DEVNET === "devnet"` that would output `true`

Reply:
Oh, now I remember, it's because one side was `"Devnet"` and the other was "Network.DEVNET", so had to lowercase one of them.

Reply:
ohh yes, wallet standard and co.